src/Helper/Wordpress.php

```diff
@@ -296,13 +296,45 @@ public static function convertTimestampToUTCDatetime( $timestamp ) {
 		return $dto;
 	}
 
+	/**
+	 * This function does no actual conversion to another timezone, it just sets the timezone to UTC
+	 *
+	 * @param $datetime
+	 *
+	 * @return DateTime
+	 * @throws \Exception
+	 */
```

Comment:
@hansmorb hast du zufällig geschaut ob die Verwendung dieser Methode denn in einer korrekten Umfeld stattfindet? Das Umfeld also nicht annimmt das hier eine umgewandelte UTC-Zeit einer lokalen Zeit rauskommt?

Reply:
Ich denke mal die Methode hat so wie sie existiert schon ihre Berechtigung, schließlich speichern wir ja alle timestamps in lokaler Zeit und nicht in UTC. Mit dieser Methode wird einfach nur ein DateTime Objekt erzeugt. Das ist dann zwar immer noch nicht in UTC aber hat zumindest eine Zeitzone (das war glaube ich für den iCalendar relevant).